src/components/RecoveryPortalWebView/RecoveryPortalWebView.js

```diff
@@ -50,10 +51,13 @@ const RecoveryPortalWebView = ({
   onRef,
   urlPath = '',
   goBackDismiss,
+  isKnowledgeBaseUrl,
 }: Props) => {
   let webViewRef;
   let canWebViewNavigateBack = false;
-  const uri = `${getEnv().RECOVERY_PORTAL_URL}/${urlPath}`;
+  // eslint-disable-next-line i18next/no-literal-string
+  const knowledgeBaseUrl = 'https://help.pillarproject.io/en/';
```

Comment:
Could we add this `knowledgeBaseUrl` as a Firebase Remote Config value please? Please use the Firebase Remote Config parameter key `knowlegebase_url`.